app/src/main/java/com/solux/flory/presentation/auth/LoginActivity.kt

```diff
@@ -13,18 +14,25 @@ class LoginActivity : BindingActivity<ActivityLoginBinding>({
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         loginBtnClick()
+        gotoSignup()
     }
 
-    private fun navigateToUserInfoActivity() {
-        Intent(this@LoginActivity, UserInfoActivity::class.java).apply {
-            addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK or Intent.FLAG_ACTIVITY_NEW_TASK)
-            startActivity(this)
+//    private fun navigateToUserInfoActivity() {
+//        Intent(this@LoginActivity, UserInfoActivity::class.java).apply {
+//            addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK or Intent.FLAG_ACTIVITY_NEW_TASK)
+//            startActivity(this)
+//        }
+//    }
+
+    private fun loginBtnClick() {
+        binding.btnLogin.setOnClickListener {
+            startActivity(Intent(this, MainActivity::class.java))
```

Comment:
로그인 성공한 순간 메인화면으로 전환되는데, 메인화면에서 뒤로가기 누르면 앱이 꺼지는게 더 적절할 것 같아요. 
```suggestion
    private fun loginBtnClick() {
        binding.btnLogin.setOnClickListener {
            Intent(this, MainActivity::class.java).apply {
                addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK or Intent.FLAG_ACTIVITY_NEW_TASK)
                startActivity(this)
            }
```